spec/services/timeline_events/create_service_spec.rb

```diff
@@ -66,5 +66,23 @@
         expect(another_submission.reload.timeline_event_owners.where(founder: another_student).first.latest).to eq(true)
       end
     end
+
+    context 'when target is an individual target with submissions from team members' do
+      let(:student) { create :student }
+      let(:another_student) { create :student, startup: student.startup }
+      let(:target) { create :target, role: Target::ROLE_STUDENT, target_group: target_group }
```

Comment:
It looks like there are two lines here repeating setup already in the root context.

```suggestion
      let(:another_student) { create :student, startup: student.startup }
```